ca/server.go

```diff
@@ -36,6 +36,10 @@ type Server struct {
 	joinTokens                  *api.JoinTokens
 	reconciliationRetryInterval time.Duration
 
+	// skipAutoUpdateRootCA, if set to true, causes the CA server to skip automatically updating
+	// the signing credentials based on changes in the memory store
+	skipAutoUpdateRootCA bool
```

Comment:
We can add a cluster watch functionality to the test CA server.  I put this in mainly because it seemed nice for the CA server to a be completely self-sufficient server, if necessary, if someone wanted to run one independently of the manager (which I agree, we are only only for tests, but the CA server would not be as usable otherwise).